tock/tock/settings/dev.py

```diff
@@ -10,9 +11,9 @@
         'ENGINE': 'django.db.backends.postgresql_psycopg2', # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
         'NAME': 'tock',                      # Or path to database file if using sqlite3.
         # The following settings are not used with sqlite3:
-        'USER': 'tock',
-        'PASSWORD': 'tock',
-        'HOST': 'localhost',                      # Empty for localhost through domain sockets or           '127.0.0.1' for localhost through TCP.
+        'USER': os.environ.get('DATABASE_USER'),
```

Comment:
@milafrerichs could the problem be a mistmatch between these env vars and the vars defined in docker-compose.yml?

```
  db:
    image: postgres
    environment:
      - POSTGRES_USER=tock
      - POSTGRES_PASSWORD=tock
      - POSTGRES_DB=tock
      - PGPASSWORD=tock
      - PGPUSER=tock
```